src/lib/list/selection-list.ts

```diff
@@ -158,7 +160,7 @@ export class MatListOption extends _MatListOptionMixinBase
   constructor(private _element: ElementRef,
               private _changeDetector: ChangeDetectorRef,
               @Optional() @Inject(forwardRef(() => MatSelectionList))
-              public selectionList: MatSelectionList) {
+              private _selectionList: MatSelectionList) {
```

Comment:
I doubt that many people are depending on this property in particular, but this is technically a breaking change.

Reply:
Good point. If really necessary we can deprecate it

Reply:
Might as well leave it, doesn't really hurt to leave it as public.

Reply:
Hm, but it doesn't really make sense from an API perspective. Also it shows up in the docs which is kind of unnecessary. I don't really mind keeping that property as it is. Just let me know.

Reply:
Can just make it docs-private

Reply:
Done.